### DISCUSSION TITLE
How to use a Lambdad transform with multiple arguments ?

### DISCUSSION THREAD

**Discussion by Maxlo24** [Project-MONAI/MONAI#3034] [Q&A] [answered]
Hi,
I have a list of coord on where to apply a same SpatialCrop transform on 2 3D images. 
In a Compose() I wanted to use a Lambda transform to call a function that takes as arguments the 2 loaded image with the keys "Image" and "seg", the crop size and the list of roi_center coords. This function then return the 2 images cropped at one of the randomly selected  roi_center.

Is it even possible to do that with Lambda or Lambdad transform ?

Thanks

**Comment by wyli** [comment, marked as answer]
It's not possible to do that with Lambda or Lambdad.  I think you should create new transforms by extending `monai.transforms.MapTransform`

**Reply by Maxlo24** [reply]
Ok, thanks
